docs/operation/troubleshooting.rst

```diff
@@ -0,0 +1,178 @@
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If you reboot the Bootstrap node and for some reason, some containers
+(especially the salt-master container) refuses to start then perform the
+following checks:
+
+- Check and ensure that the **MetalK8s ISO** is mounted properly.
+
+  .. parsed-literal::
+
+     [root@bootstrap vagrant]# mount | grep /srv/scality/metalk8s-|release|
+     /home/centos/metalk8s.iso on /srv/scality/metalk8s-|release| type iso9660 (ro,relatime)
+
+
+- If the ISO is unmounted, run the following command which will check the
+  the status of the ISO file and remount it automatically.
+
+  .. code-block:: shell
+
+     [root@bootstrap vagrant]# salt-call state.sls metalk8s.archives.mounted saltenv=metalk8s-|release|
+      Summary for local
+      ------------
+      Succeeded: 3
+      Failed:    0
+
+Bootstrap fails and console log is unscrollable
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If during a MetalK8s installation, the Bootstrap process fails and the console
+output is unscrollable then you can consult the Bootstrap logs in
+``/var/log/metalk8s-bootstrap.log``.
+
+Account Administration Errors
++++++++++++++++++++++++++++++
+
+Forgot the MetalK8s GUI password
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If you forget the MetalK8s GUI username and/or password combination,
+follow
+:ref:`this procedure <Administering-MetalK8s-GUI-Kubernetes-API-and-Salt-API>`
+to reset or change it.
+
+General Kubernetes Resource Errors
+++++++++++++++++++++++++++++++++++
+
+Pod status shows "CrashLoopBackOff"
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If after a MetalK8s installation, you notice some Pods are in a state of
+"CrashLoopBackOff", then it means pods are crashing because they start up then
+immediately exit, thus Kubernetes restarts them and the cycle continues.
+To get possible clues about this error, run the following commands and inspect
+the output.
+
+.. code-block:: shell
+
+   [root@bootstrap vagrant]# kubectl -n kube-system describe pods <pod name>
+    Name:                 <pod name>
+    Namespace:            kube-system
+    Priority:             2000000000
+    Priority Class Name:  system-cluster-critical
+
+Persistent Volume Claim(PVC) stuck in "Pending" state
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If after provisioning a Volume for a Pod (e.g. Prometheus) and the PVC still
+hangs in a **Pending** state, then try checking the following:
+
+- Check that the volumes have been provisioned and are in a **Ready** state:
+
+  .. code-block:: shell
+
+     kubectl describe volume <volume-name>
+     [root@bootstrap vagrant]# kubectl describe volume test-volume
+      Name:         <volume-name>
+      Status:
+        Conditions:
+          Last Transition Time:  2020-01-14T12:57:56Z
+          Last Update Time:      2020-01-14T12:57:56Z
+          Status:                True
+          Type:                  Ready
+
+- Check that a corresponding PersistentVolume exist:
```

Comment:
Another thing worth checking is that the PVC constraints match the PV (min size, labels, storage class).